JS-Drum-Kit/app.js

```diff
@@ -0,0 +1,34 @@
+const keys = document.querySelectorAll('.key');
+
+
+function play(keyCode) {
+	const audio = document.querySelector(`audio[data-key="${keyCode}"]`);
+	if(!audio) return;
+	audio.currentTime = 0;
+    audio.play();
+    const key = document.querySelector(`.key[data-key="${keyCode}"]`);
+    key.classList.add('playing');
+};
+
+// Play drum by keyboard press
+function playDrum(e) {
+    play(e.keyCode)
+    audio.play();
```

Comment:
You don't need a call to this function, please pay attention to the browser console

Reply:
update this code